<?xml version="1.0" encoding="UTF-8"?>
<testrun name="UndoManagerUnitTests" project="UndoManager" tests="3" started="3" failures="0" errors="1" ignored="0">
  <testsuite name="undo.UndoManagerUnitTests" time="0.018">
    <testcase name="testBufferCreation" classname="undo.UndoManagerUnitTests" time="0.007"/>
    <testcase name="testBufferAdd" classname="undo.UndoManagerUnitTests" time="0.005"/>
    <testcase name="testUndoSingleItem" classname="undo.UndoManagerUnitTests" time="0.006">
      <error>java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
	at java.util.LinkedList.entry(LinkedList.java:382)
	at java.util.LinkedList.get(LinkedList.java:332)
	at undo.UndoManagerImpl.currentChange(UndoManagerImpl.java:92)
	at undo.UndoManagerUnitTests.testUndoSingleItem(UndoManagerUnitTests.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:622)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:243)
	at junit.framework.TestSuite.run(TestSuite.java:238)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)

</error>
    </testcase>
  </testsuite>
</testrun>
